web-ui/src/main/resources/catalog/components/search/searchfiltertag/partials/searchFilterTagsTemplate.html

```diff
@@ -12,7 +12,10 @@
        ng-repeat="filter in currentFilters"
        ng-click="removeFilter(filter)">
       <span class="fa fa-times text-danger delete-icon"></span>
-      <strong class="text-no-wrap text-uppercase" translate>{{filter.key}}</strong>
+      <strong class="text-no-wrap text-uppercase" translate>
+        <span ng-if="filter.key!='any'">{{('facet-' + filter.key)}}</span>
```

Comment:
The `translate` directive should be added here, otherwise the translation does not happen.